mmcv/runner/hooks/lr_updater.py

```diff
@@ -65,10 +67,10 @@ def _set_lr(self, runner, lr_groups):
                                        lr_groups):
                 param_group['lr'] = lr
 
-    def get_lr(self, runner, base_lr):
+    def get_lr(self, runner: BaseRunner, base_lr: float):
```

Comment:
We can remove the type hint of runner because it will cause a circular import.

Reply:
I have fixed it.